src/swappers/MagicApeLevSwapper.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+// solhint-disable avoid-low-level-calls
+pragma solidity >=0.8.0;
+
+import "BoringSolidity/interfaces/IERC20.sol";
+import "BoringSolidity/libraries/BoringERC20.sol";
+import "libraries/SafeApprove.sol";
+import "interfaces/IBentoBoxV1.sol";
+import "interfaces/ILevSwapperV2.sol";
+import "interfaces/IERC4626.sol";
+import "interfaces/IGmxGlpRewardRouter.sol";
+import "interfaces/IGmxVault.sol";
+
+contract MagicApeLevSwapper is ILevSwapperV2 {
```

Comment:
One nit pick here, why don't we make this an arbitrary ERC4626 swapper

Reply:
you're right, I'll adapt it so it's generic